src/app/portlets/dot-edit-page/layout/dot-edit-layout-grid/dot-edit-layout-grid.component.ts

```diff
@@ -119,59 +91,53 @@ export class DotEditLayoutGridComponent implements OnInit {
     }
 
     /**
-     * Event fired when the grad of a container ends, remove empty rows if any.
-     * @constructor
+     * Event fired when the drag of a container ends, remove empty rows if any.
+     *
      */
     onDragStop(): void {
         this.deleteEmptyRows();
     }
 
     /**
-     * Transform the grid data into DotLayoutBody.
-     * @returns {DotLayoutBody}
+     * Return ng-grid model.
+     * @returns {DotLayoutGridBox[]}
      */
-    getLayoutBody(): DotLayoutBody {
-        return this.dotEditLayoutGridService.getDotLayoutBody(this.gridBoxes);
+    getGrid(): DotLayoutGridBox[] {
+        return this.grid;
```

Comment:
Again, this will not transform nothing, whoever use this component will get the model as it is, if they need to transform it is their problem.